src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -61,12 +64,19 @@ public Events(FlareBot bot) {
     @Override
     public void onReady(ReadyEvent event) {
         FlareBot.getInstance().latch.countDown();
+        new Timer().schedule(new TimerTask() { //If you're not ok with this in the ready event just say so.
+            @Override
+            public void run() {
+                spamMap = new HashMap<>(); //According to stack overflow it's better to just create a new hashmap and let the garbage collector get the other one.
```

Comment:
ConcurrentHashMap, and no it's not, and we have the expiring map implemented too